crime-applications-adaptor/src/main/resources/schemas/maat-caa-contract.json

```diff
@@ -0,0 +1,172 @@
+{
+  "type": "object",
+  "description": "Means assessment application object",
+  "properties": {
+    "statusReason": {
+      "type": "string",
+      "default": "current",
+      "description": "Currently not populated."
+    },
+    "solicitorName": {
+      "type": "string",
+      "description": "The full name of the solicitor."
+    },
+    "repId": {
+      "type": "integer",
+      "nullable": true,
+      "description": "Currently not populated."
+    },
+    "maatRef": {
```

Comment:
Is it possible to move this to a folder for clear  demarkation between crime-apply schemas and MAAT application schema?

Reply:
Yes, that's a great idea, I will do that

Reply:
Have moved our schema to `crime-applications-adaptor/src/main/resources/schemas/crimeapplicationsadaptor`

Reply:
Can we move the schemas related to meat-caa to a folder crime-applications-adaptor/src/main/resources/schemas/maatapplication